cmd/build.go

```diff
@@ -98,8 +99,16 @@ func build(config *buildCommandConfig) error {
 			return err
 		}
 		cmdArgs = append(cmdArgs, options...)
+	}
 
+	labels, err := getLabels(config)
+	if err != nil {
+		return errors.Errorf("%v", err)
```

Comment:
Is there a reason why you are wrapping the `err` in another error without changing the message?